docs/testing.md

```diff
@@ -92,34 +88,97 @@ being used in integration tests, since they delete the test namespaces.
+## End-to-End
 
-### Upload Operator Image
+The e2e tests are meant to test acceptance scenarios. They are written from an end user perspective.
+They are split into two types, 'cli' and 'kube'.
 
-Template rendering for BOSH jobs is done at deployment time by the operator
-binary. Therefore the operator docker image needs to be made available to
-Kubernetes cluster.
+The e2e CLI test exercise different command line options and commands which don't need a running Kubernetes, like template rendering.
+The CLI tests build the operator binary themselves.
 
-For running integration tests locally against minikube, we switch to minikubes
-docker daemon, build the binary, copy it to a docker image and finally start
-the tests:
+The second type of e2e tests use `helm` to install the CF operator into the k8s cluster and use the files from `docs/examples` for testing.
 
-    eval `minikube docker-env`
-    bin/build; bin/build-nobuild-image
-    bin/test-integration
 
-The image source can be configured by these environment variables:
+# Running tests in minikube
 
-    DOCKER_IMAGE_ORG
-    DOCKER_IMAGE_REPOSITORY
-    DOCKER_IMAGE_TAG
+Prior to running the tests, is highly recommended to execute the following steps:
 
-## End-to-End
+_**Note**_: This steps are aimed to run on a `minikube` environment.
 
-The e2e tests are meant to test acceptance scenarios. They are written from an end user perspective.
-They are split into two types, 'cli' and 'kube'.
+1. Start `minikube`
 
-The e2e CLI test exercise different command line options and commands which don't need a running Kubernetes, like template rendering.
-The CLI tests build the operator binary themselves.
+    ```bash
+    minikube start
+    ```
 
-The second type of e2e tests use `helm` to install the CF operator into the k8s cluster and use the files from `docs/examples` for testing.
+2. Switch to minikube docker daemon
+
+    ```bash
+    eval $(minikube docker-env)
+    ```
+
+    _**Note**_: Template rendering for BOSH jobs is done at deployment time by the operator
+    binary. Therefore the operator docker image needs to be made available to
+    Kubernetes cluster.
+
+
+3. Export the `CF_OPERATOR_WEBHOOK_SERVICE_HOST` env variable
+
+    ```bash
+    export CF_OPERATOR_WEBHOOK_SERVICE_HOST=$(minikube ssh -- "cat /etc/resolv.conf | grep nameserver | awk '{ printf \$2 }'")
+    ```
+
+    _**Note**_: You can also find the correct IP, by running `ip addr`. The IP address under `vboxnet1` is the IP that you need.
+
+4. Export the `OPERATOR_TEST_STORAGE_CLASS` env variable
+
+    ```bash
+    export OPERATOR_TEST_STORAGE_CLASS=standard
+    ```
+
+    _**Note**_: Require for the PVC test creation, in minikube.
+
+5. Ensure `GO111MODULE` is set
+
+    ```bash
+    export GO111MODULE=on
+    ```
+
+6. Build the `cf-operator` binary
+    ```bash
+    bin/build
+    ```
+
+7. Build the `cf-operator` docker image
+    ```bash
+    bin/build-image
+    ```
+
+# Make targets
```

Comment:
There is a nice table about those targets in `docs/development.md`. Maybe we should have 'makefile.md` and link it from both places

Reply:
That table should belong to `testing.md`, I will move it there. While every related item to tests, should be in that file.